pages/employee-owned-trust.tsx

```diff
@@ -27,10 +27,17 @@ const EmployeeOwnedTrustPage: NextPage<EmployeeOwnedTrustPageProps> = ({
     jobsAvailable,
     content,
 }) => {
-    const benefits =
-        content.itemsCollection.items[0].benefitsListCollection.items.map(
-            (item: any) => item.benefitName,
-        );
+    const benefitCollection = content.itemsCollection.items.find(
+        (obj: any) => obj.__typename === 'Benefits',
+    );
+
+    const benefits = benefitCollection.benefitsListCollection.items.map(
+        (item: any) => item.benefitName,
+    );
+
+    const voiceOfChangeCollection = content.itemsCollection.items.find(
+        (obj: any) => obj.__typename === 'VoiceOfChange',
+    );
```

Comment:
This little 'find' function could be abstracted maybe? Should I create a utilities function file for this? Might not be a major issue